### PR TITLE
Traits YML cleanup and rearranging

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
image

**Changelog**
:cl:
- fix: Fixed duplicate traits in the character Traits tab. Check to make sure that your selected traits are still correct!

### CODE REVIEW

**Comment by Rainbeon** [comment]
There are still 2 sets of Frontal Lisp in your screenshot, one free and one which costs 2 points

**Comment by Dutch-VanDerLinde** [comment]
> There are still 2 sets of Frontal Lisp in your screenshot, one free and one which costs 2 points

Fixed

File: Resources/Locale/en-US/_RMC14/traits/traits.ftl [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+rmc-trait-french-accent-desc = You come from space france
+
+rmc-trait-mobster-accent-name = Mobster accent
+rmc-trait-mobster-accent-desc = ey gabagool fugeddaboutit I'm walkin here 
+
+rmc-trait-spanish-accent-name = Spanish accent
+rmc-trait-spanish-accent-desc = You come from space spain
+
+rmc-trait-scottish-accent-name = Scottish accent
+rmc-trait-scottish-accent-desc = Fer tha folk who come fae a Hielan clan.
```
Comment (DrSmugleaf) [inline Resources/Locale/en-US/_RMC14/traits/traits.ftl:16]:
```suggestion
rmc-trait-category-inconveniences = Inconveniences

rmc-trait-russian-accent-name = Russian accent
rmc-trait-russian-accent-desc = You come from space Russia

rmc-trait-french-accent-name = French accent
rmc-trait-french-accent-desc = You come from space France

rmc-trait-mobster-accent-name = Mobster accent
rmc-trait-mobster-accent-desc = ey gabagool fugeddaboutit I'm walkin here 

rmc-trait-spanish-accent-name = Spanish accent
rmc-trait-spanish-accent-desc = You come from space Spain

rmc-trait-scottish-accent-name = Scottish accent
rmc-trait-scottish-accent-desc = Fer tha folk who come fae a Hielan clan.

```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!
This will remove the currently chosen traits for anyone that has them but I guess that's fine, I can try to do a database migration if I remember to

**Pull request merged**